bin/summarize-user-events

```diff
@@ -28,12 +28,9 @@ class SummarizeUserEvents
   end
```

Comment:
I hadn't seen this script before, so leaving comments I would have left if it was new! I would see if we can use the optparse library to do CLI args and name them with flags, I think it's more discoverable than hardcoded order!

Reply:
Agreed, this'll come eventually